manifests/cluster-install/workflow-controller-rbac/workflow-controller-clusterrole.yaml

```diff
@@ -52,3 +52,13 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  - serviceaccounts/finalizers
```

Comment:
I don't think you need to add /finalizers for the core types.  /finalizers was originally added for workflow CRD to support openshift, but I don't think it is necessary for core types.